data-prepper-plugins/peer-forwarder/src/main/java/com/amazon/dataprepper/plugins/prepper/peerforwarder/PeerForwarder.java

```diff
@@ -21,15 +21,22 @@
 import java.util.ArrayList;
 import java.util.Collection;
 import java.util.HashMap;
+import java.util.LinkedList;
 import java.util.List;
 import java.util.Map;
+import java.util.concurrent.CompletableFuture;
 import java.util.concurrent.ConcurrentHashMap;
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
 
 @DataPrepperPlugin(name = "peer_forwarder", type = PluginType.PREPPER)
 public class PeerForwarder extends AbstractPrepper<Record<ExportTraceServiceRequest>, Record<ExportTraceServiceRequest>> {
-    public static final String FORWARD_REQUEST_LATENCY_PREFIX = "forwardRequestLatency";
-    public static final String FORWARD_REQUEST_SUCCESS_PREFIX = "forwardRequestSuccess";
-    public static final String FORWARD_REQUEST_ERRORS_PREFIX = "forwardRequestErrors";
+    public static final String REQUESTS = "requests";
+    public static final String LATENCY = "latency";
+    public static final String ERRORS = "errors";
+    public static final String DESTINATION = "destination";
```

Comment:
Typically I'd say, of course don't change the values to avoid breaking dashboards and queries, but in this case it's quite a dramatic change in that dashboards are going to have to be updated regardless of if the keys are kept the same or not. Previously the metric would look something like `entry_pipeline_peer_forwarder_forwardRequestSuccess_123_456_7_8_total` (note the IP in the metric name), whereas now it's just `entry_pipeline_peer_forwarder_requests_total`. 

This change is to get away from requiring users to type in specific IP addresses to create dashboards, so naturally the dashboards will have to change (hopefully just this one time).